app/test_main.py

```diff
@@ -1,3 +1,48 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAgeShouldReturnExpectedResult:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_result",
+        [
+            (-1, -1, [0, 0]),
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17]),
+            (5, 10, [0, 0]),
```

Comment:
Add test cases that would be variable enough in the future, so that `expected_result` would have a pair of values other than `0`